script-base.js

```diff
@@ -409,7 +558,7 @@ Generator.prototype.addSocialConnectionFactory = function (javaDir, importPackag
  *
  */
 Generator.prototype.addMainCSSStyle = function(style, comment) {
-    // Not Working this.useSass -> Undifined
+    // Not Working this.useSass -> Undifined --> this in the modules context would be jhipsterFunc
```

Comment:
@deepu105  you mean that is the module who give the useSass constant? I need to add a new argument like this:

` function(style, comment, isUseSass)`